webdriver/OWNERS

```diff
@@ -0,0 +1,5 @@
+@andreastt
+@lukeis
+@AutomatedTester
+@shs96c
+@emalasky
```

Comment:
He probably merged a commit that touched the webdriver test suite at some point.